### PR TITLE
TATB : 06 Change the JSON file with Proper Format and get Options randomly in the quiz.

### PR DESCRIPTION
**Pull request by SohamDevlani**
-Change the JSON file with Proper Format
    -Change path everywhere to get data from JSON file.
    -Get Options in the quiz randomly

### CODE REVIEW

File: src/app/component/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -9,25 +9,19 @@ import { QuestionService } from '../../service/question.service';
   styleUrls: ['./dashboard.component.css']
 })
 export class DashboardComponent implements OnInit,OnDestroy {
-  quizData = quizData;
+  quizData = {...quizData};
   title: any;
   arrayIndex=0;
   image: any;
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts:15]: what is image is used for ?

File: src/app/component/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -9,25 +9,19 @@ import { QuestionService } from '../../service/question.service';
...
   description: any;
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts:16]: description is used for ?

File: src/app/component/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -36,19 +30,10 @@ export class DashboardComponent implements OnInit,OnDestroy {
...
   ){}
   
-  mapJSONData(){
-    // this.title = this.quizData?.QuizType?.title;
-    // this.image = this.quizData?.QuizType;
-    // this.description = this.quizData?.QuizType;
-    // console.log("title",this.title)
-    // console.log("image",this.image)
-    // console.log("description",this.description)
+  formatData() {
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts:33]: Could you please remove this function as there is nothing in this

File: src/app/component/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -36,19 +30,10 @@ export class DashboardComponent implements OnInit,OnDestroy {
...
+    console.log(this.QuizType); 
   }
-  // next(){
-  //   this.imagesindex++;
-  //   this.imagessindex++;
-  //   this.imagesssindex++;
-  // }
+
   quizname(title:any){
     console.log(title);
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts:38]: Please remove console from here

File: src/app/component/dashboard/dashboard.component.ts [hunk 5]
```diff
@@ -36,19 +30,10 @@ export class DashboardComponent implements OnInit,OnDestroy {
...
     this.questionService.selectedQuizType = title;
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts:39]: I guess service is not in use please remove this

File: src/app/component/quiz/quiz.component.ts [hunk 6]
```diff
@@ -67,6 +54,7 @@ export class Quizcomponent implements OnInit, OnDestroy {
   timer: any;
   selectedQuizType: any;
   radioValue: any;
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:56]: Could you please check and remove if not in use

File: src/app/component/quiz/quiz.component.ts [hunk 7]
```diff
@@ -20,28 +20,15 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
...
-    'Q-8.',
-    'Q-9',
-    'Q-10',
-  ];
+
   question!: any;
-  options!: any;
-  title!: any;
   dialogData = { ...dialogData };
   private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:34]: Please do code indentation

File: src/app/component/quiz/quiz.component.ts [hunk 8]
```diff
@@ -215,17 +210,18 @@ export class Quizcomponent implements OnInit, OnDestroy {
         this.points = this.points += 1;
         this.correctanswer++;
       } else if (!selectedAnswer && correctOptions) {
-        debugger;
         this.points = this.points -= 0.25;
         this.inCorrectanswer++;
       }
-      if (questionindex === this.question.length) {
+      debugger;
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:216]: Could you please remove debugger from here

File: src/app/component/result/result.component.ts [hunk 9]
```diff
@@ -8,7 +8,7 @@ import quizData from 'src/assets/json/data.json';
   styleUrls: ['./result.component.css'],
 })
 export class ResultComponent implements OnInit, OnDestroy {
-  quizData: any = { ...quizData };
+  quizData = { ...quizData };
   isQuizCompleted: any;
```
Comment (PoojanPatelJT) [inline src/app/component/result/result.component.ts:12]: what is isQuizCompleleted is used for ?

File: src/app/component/result/result.component.ts [hunk 10]
```diff
@@ -8,7 +8,7 @@ import quizData from 'src/assets/json/data.json';
...
   userName: any;
```
Comment (PoojanPatelJT) [inline src/app/component/result/result.component.ts:13]: Could you please check and remove userName if not in  use

File: src/app/component/result/result.component.ts [hunk 11]
```diff
@@ -8,7 +8,7 @@ import quizData from 'src/assets/json/data.json';
...
   points: any;
```
Comment (PoojanPatelJT) [inline src/app/component/result/result.component.ts:14]: Same with this

Comment (PoojanPatelJT) [reply src/app/component/result/result.component.ts:14]: correctAnswer: any;
  inCorrectAnswer: any; 
  
  Please remove this function

File: src/app/component/startquiz/startquiz.component.html [hunk 12]
```diff
@@ -4,7 +4,7 @@
     <img class="card-img-top" src="assets\img\rules.png" alt="">
     <h2 class="card-title text-center"> Welcome to Quiz App</h2>
      <h4 class="card-subtitle"> Rules:</h4>
-     <ol [innerHTML]="rules">
+     <ol [innerHTML]="rules.rules">
```
Comment (PoojanPatelJT) [inline src/app/component/startquiz/startquiz.component.html:7]: Could you please change variable name

File: src/app/component/startquiz/startquiz.component.ts [hunk 13]
```diff
@@ -24,11 +24,8 @@ export class StartquizComponent implements OnInit {
   }
 
   mapJSONData() {
-    if (this.questionService.selectedQuizType) {
-      this.selectedQuizType = this.questionService.selectedQuizType;
-    }
-   this.rules = this.quizData[this.selectedQuizType]?.rules;
-   console.log("this.rules ----------",this.rules)
+    this.selectedQuizType = this.activeRoute.snapshot.queryParams['quiz'];
+   this.rules = this.quizData.Quiz.find((data)=> data.quizId == this.selectedQuizType );
```
Comment (PoojanPatelJT) [inline src/app/component/startquiz/startquiz.component.ts:28]: Please add null check here

File: src/app/component/result/result.component.ts (hunk 11 above)
Comment (PoojanPatelJT) [reply src/app/component/result/result.component.ts:14]: Please remove questionservice

**Review by PoojanPatelJT — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.ts [hunk 14]
```diff
@@ -20,53 +20,31 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
...
-  findIndex: any;
-  filter: any;
   filterquestionsindex: any;
-  checkIfselctedanswer: any;
-  userName: any;
   timer: any;
   selectedQuizType: any;
   radioValue: any;
+  options: string | number | undefined;
+  positivePoints: any;
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:44]: correctAnswer: number = 0;
  wrongAnswer: number = 0; 
  Why you have declared this  ? As in above i can see those variable

File: src/app/component/quiz/quiz.component.ts [hunk 15]
```diff
@@ -99,11 +77,27 @@ export class Quizcomponent implements OnInit, OnDestroy {
...
+    this.question = [...data];
     this.question = this.question?.sort(() => Math.random() - 0.67);
-    this.question = [...this.question?.splice(0, 11)];
-    this.options = this.question[this.questionindex]?.options;
-    this.timer = this.quizData[this.selectedQuizType]?.timer;
+    this.question = [...this.question?.splice(0, this.numberOfQuestions)];
+    this.options = this.question.map((question: any) =>
+      question.options.sort(() => Math.random() - 0.69)
+    );
+    this.timer = this.quizData.Quiz.find(
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:92]: Can we do all this in one ?

**Review by PoojanPatelJT — CHANGES_REQUESTED** [review]

File: src/app/component/result/result.component.ts (hunk 10 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/result/result.component.ts:12]: Yes i removed it.

File: src/app/component/result/result.component.ts (hunk 11 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/result/result.component.ts]: i removed all unnecessary functions

File: src/app/component/startquiz/startquiz.component.html (hunk 12 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/startquiz/startquiz.component.html]: i changed variable name.

File: src/app/component/dashboard/dashboard.component.ts (hunk 2 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: It wasnt used anywhere so removed it.

Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: It wasnt used anywhere hence removed it

File: src/app/component/dashboard/dashboard.component.ts (hunk 3 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i removed it

File: src/app/component/dashboard/dashboard.component.ts (hunk 4 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i removed it

File: src/app/component/dashboard/dashboard.component.ts (hunk 5 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i removed it

File: src/app/component/quiz/quiz.component.ts (hunk 6 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts:42]: Yes i removed all unnecessary declarations

File: src/app/component/quiz/quiz.component.ts (hunk 7 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts]: Done

File: src/app/component/quiz/quiz.component.ts (hunk 8 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts]: yes i removed it

File: src/app/component/result/result.component.ts (hunk 9 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/result/result.component.ts]: it wasnt used anywhere so removed it

File: src/app/component/startquiz/startquiz.component.ts (hunk 13 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/startquiz/startquiz.component.ts]: i added the null check

File: src/app/component/dashboard/dashboard.component.ts [hunk 16]
```diff
@@ -1,57 +1,25 @@
...
-  //  imagesssindex=2;
-
-    
-   
    ngOnInit(): void {
-    console.log(this.quizData.QuizType)
-    this.QuizType = this.quizData?.QuizType;
-
-    // this.mapJSONData();
+    this.QuizType = this.quizData.Quiz;
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.ts]: please use camelCase for the varaibles

File: src/app/component/quiz/quiz.component.ts [hunk 17]
```diff
@@ -20,53 +20,31 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
...
-  ];
   question!: any;
-  options!: any;
-  title!: any;
   dialogData = { ...dialogData };
   private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
-
-  selectedOption!: boolean;
-
   questionindex = 0;
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts]: please give proper datatypes for questionIndex, counter, intervals$ variables

File: src/app/component/quiz/quiz.component.ts [hunk 18]
```diff
@@ -99,11 +77,27 @@ export class Quizcomponent implements OnInit, OnDestroy {
...
+    )?.timer;
+    this.positivePoints = this.quizData.Quiz.find(
+      (data) => data.quizId == this.selectedQuizType
+    )?.PositivePoints;
+    this.negativePoints = this.quizData.Quiz.find(
+      (data) => data.quizId == this.selectedQuizType
+    )?.NegativePoints;
 
     const formArray = this.myForm.controls['form'] as FormArray;
     this.question.forEach((item: any) => {
```
Comment (PoojanPatelJT) [inline src/app/component/quiz/quiz.component.ts:100]: Please use camelCase for all varaibles

File: src/app/component/dashboard/dashboard.component.ts [hunk 19]
```diff
@@ -1,57 +1,25 @@
...
-
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.css']
 })
 export class DashboardComponent implements OnInit,OnDestroy {
-  quizData = quizData;
+  quizData = {...quizData};
   title: any;
```
Comment (charmijtdev) [inline src/app/component/dashboard/dashboard.component.ts]: Please give proper format like string

File: src/app/component/dashboard/dashboard.component.ts [hunk 20]
```diff
@@ -1,57 +1,25 @@
...
   arrayIndex=0;
-  image: any;
-  description: any;
-  QuizType:any = [0,1,2];
+  QuizType:any;
```
Comment (charmijtdev) [inline src/app/component/dashboard/dashboard.component.ts]: here too and please give name like quizType:string

File: src/app/component/dashboard/dashboard.component.ts [hunk 21]
```diff
@@ -1,57 +1,25 @@
...
-    // console.log("image",this.image)
-    // console.log("description",this.description)
-  }
-  // next(){
-  //   this.imagesindex++;
-  //   this.imagessindex++;
-  //   this.imagesssindex++;
-  // }
+
   quizname(title:any){
```
Comment (charmijtdev) [inline src/app/component/dashboard/dashboard.component.ts]: Please use camel case name standard and please change like routeToQuiz

File: src/app/component/quiz/quiz.component.ts [hunk 22]
```diff
@@ -20,53 +20,31 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
...
   private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
-
-  selectedOption!: boolean;
-
   questionindex = 0;
   counter = 30;
   interval$: any;
-  validate: any;
   points: number = 0;
   correctanswer: number = 0;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.ts:34]: Please check again  
correctAnswer: number = 0;
  wrongAnswer: number = 0;

File: src/app/component/quiz/quiz.component.ts (hunk 14 above)
Comment (charmijtdev) [reply src/app/component/quiz/quiz.component.ts:41]: This is uesd three time Please check 
correctanswer: number = 0;

File: src/app/component/quiz/quiz.component.ts (hunk 17 above)
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.ts]: could you please change name to questionindex to questionIndex

File: src/app/component/quiz/quiz.component.ts [hunk 23]
```diff
@@ -20,53 +20,31 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
...
-  startquiz: any;
-  answerlistArray: any = [];
-  submittedAnswer: any[] = [];
-  isQuizCompleted: boolean = false;
-  findIndex: any;
-  filter: any;
   filterquestionsindex: any;
-  checkIfselctedanswer: any;
-  userName: any;
   timer: any;
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.ts]: Here please remove timer as counter is used upper and give type

File: src/app/component/startquiz/startquiz.component.ts [hunk 24]
```diff
@@ -9,8 +9,8 @@ import quizData from '../../../assets/json/data.json';
   styleUrls: ['./startquiz.component.css']
 })
 export class StartquizComponent implements OnInit {
-  quizData:any = quizData;
-  rules: any;
+  quizData = {...quizData};
+  instruction: any;
   selectedQuizType: any;
```
Comment (charmijtdev) [inline src/app/component/startquiz/startquiz.component.ts]: Please give type string

File: src/assets/json/data.json [hunk 25]
```diff
@@ -1,1107 +1,1110 @@
...
+  "Quiz": [
     {
-      "quizId": "htmlcss",
+      "quizId": "Html/CSS",
+      "rules": "<li class=\"container\">There will be a Negative Marking.</li><ul><li><b>Correct Answer </b> gives you <b>1 Point.</b></li><li><b>Incorrcet Answer </b> Deduct your <b> -0.25 Point.</b></li></ul><li>You will be given the time for each question.</li><ul><li>You have <b> 30 secound </b> for each questions.</li></ul><li>After Completion of time for each question , The question will be change automatically.</li><li>If you want to go for next question before time ending then click on the <b>Next button</b></li>",
+      "timer": 30,
+      "PositivePoints": 10,
+      "NegativePoints": 1,
+      "numberOfQuestions": "3",
+      "type": "radio",
```
Comment (charmijtdev) [inline src/assets/json/data.json:10]: what is use for type here ?

File: src/assets/json/data.json [hunk 26]
```diff
@@ -1,1107 +1,1110 @@
...
+              "label": "@font-face",
+              "type": "radio"
             }
           ],
-          "answer": ["c"]
+          "answer": { "id": 3 }
         },
         {
-          "id": "Q-11.",
+          "questionId": "Q-11.",
```
Comment (charmijtdev) [inline src/assets/json/data.json]: what is use of this question id ?

**Review by charmijtdev — CHANGES_REQUESTED** [review]

File: src/app/component/quiz/quiz.component.ts (hunk 14 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts:41]: yes i have removed unnecessary variables

File: src/assets/json/data.json (hunk 26 above)
Comment (PrabhatJTDeveloper) [reply src/assets/json/data.json]: There was no use of questionId hence removed

File: src/assets/json/data.json (hunk 25 above)
Comment (PrabhatJTDeveloper) [reply src/assets/json/data.json:10]: type is given for future enhancement as we could change the option type from radio button to checbox.

File: src/app/component/startquiz/startquiz.component.ts (hunk 24 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/startquiz/startquiz.component.ts]: yes i gave

File: src/app/component/quiz/quiz.component.ts (hunk 23 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts]: i couldnt remove timer as data of timer is coming from json file but counter was able to be removed so removed it

File: src/app/component/quiz/quiz.component.ts (hunk 17 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts]: Yes i changed the name

File: src/app/component/quiz/quiz.component.ts (hunk 22 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts:34]: yes i have removed it

File: src/app/component/dashboard/dashboard.component.ts (hunk 21 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i have given camelcase name and changed name

File: src/app/component/dashboard/dashboard.component.ts (hunk 20 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i have given type and also changed the name

File: src/app/component/dashboard/dashboard.component.ts (hunk 19 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i gave proper format to every variable

File: src/app/component/quiz/quiz.component.ts (hunk 18 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts:100]: yes i used camel case for all variables

File: src/app/component/quiz/quiz.component.ts (hunk 17 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.ts]: yes i gave proper data types

File: src/app/component/dashboard/dashboard.component.ts (hunk 16 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.ts]: yes i used camelcase for the variables

File: src/app/component/allresults/allresults.component.html [hunk 27]
```diff
@@ -0,0 +1,62 @@
+<div class="col text-center mb-3" style="margin-top: 50px">
```
Comment (charmijtdev) [inline src/app/component/allresults/allresults.component.html]: Could you plese remove this inline css ?

File: src/app/component/allresults/allresults.component.ts [hunk 28]
```diff
@@ -0,0 +1,68 @@
...
+  userData: any;
+  submittedData: any;
+  initialData:number = 8;
+  showMoreData:any;
+  allResultData: any;
+
+  constructor(public questionService: QuestionService) {}
+
+  ngOnInit(): void {
+    this.userName = this.questionService?.userName;
```
Comment (charmijtdev) [inline src/app/component/allresults/allresults.component.ts]: No need to check here ?

File: src/app/component/dashboard/dashboard.component.html [hunk 29]
```diff
@@ -1,94 +1,40 @@
...
-                {{descriptions[imagesssindex]}}
-              </p>
-              <div class="text-center">
-                <button type="button" class="btn btn-primary" (click)="quizname()">Start Quiz</button>
-              </div>
+<div class="container">
+  <div class="row">
+    <div class="min-vh-100 d-flex align-items-center">
+      <div class="card   mt-5" id="card">
+        <div class="card-body" style="padding: 0px">
```
Comment (charmijtdev) [inline src/app/component/dashboard/dashboard.component.html]: Please remove inline css

File: src/app/component/quiz/quiz.component.html [hunk 30]
```diff
@@ -45,12 +44,12 @@
...
                     </div>
-                    <!-- <canvas id="stage" width="200" height="300">
-            
-              </canvas> -->
 
+<<<<<<< HEAD
+=======
 
-              <ng-container *ngFor="let card of question[this.questionindex].options">
+>>>>>>> c2bb962d932820cb3282424f75c3ea93d58f0eac
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.html]: Please also remove this

File: src/app/component/quiz/quiz.component.html [hunk 31]
```diff
@@ -45,12 +44,12 @@
                         >
                       </div>
                     </div>
-                    <!-- <canvas id="stage" width="200" height="300">
-            
-              </canvas> -->
 
+<<<<<<< HEAD
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.html]: remove thsi confict

File: src/app/component/quiz/quiz.component.ts [hunk 32]
```diff
@@ -157,75 +149,69 @@ export class Quizcomponent implements OnInit, OnDestroy {
...
-    if (!this.FormArray.at(questionindex).get('timer')?.disabled) {
-      if (questionindex === this.question.length) {
-        this.isQuizCompleted = true;
-      }
+  answer(questionIndex: number, correctOptions: string) {
+    if (!this.FormArray.at(questionIndex).get('timer')?.disabled) {
       let selectedAnswer =
-        this.question[questionindex].answer?.[0] == correctOptions;
+        this.question[questionIndex].answer.id == correctOptions;
       if (selectedAnswer && correctOptions) {
```
Comment (charmijtdev) [inline src/app/component/quiz/quiz.component.ts]: No need to create a comparision your can direct check this condition here

File: src/app/component/result/result.component.html [hunk 33]
```diff
@@ -1,88 +1,49 @@
-<div class="card shadow" id="card" style="margin-top: 80px">
-  <div class="card-body" style="padding: 0px">
-    <div class="row g-0">
-      <div class="col text-center mb-3 mt-4">
-        <span class="fs-3">Result</span>
+<div class="col-lg-5 col-md-7 col-sm-12 col-xs-12" style="margin: 75px auto; float: none; margin-bottom: 10px;">
```
Comment (charmijtdev) [inline src/app/component/result/result.component.html]: Please remove this inline css here

File: src/app/component/result/result.component.html [hunk 34]
```diff
@@ -1,88 +1,49 @@
...
-        </div>
-      </div>
     </div>
-    <div *ngIf="!submittedData.length">
-       <h1>
-        No Quiz Played yet
-       </h1>
+    <div class="container">
+    <div class="row">
+      <div class="text-center" style="margin-bottom: 30px;">
```
Comment (charmijtdev) [inline src/app/component/result/result.component.html]: Here also please remove this

**Review by charmijtdev — APPROVED** [review]

File: src/app/component/allresults/allresults.component.html (hunk 27 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/allresults/allresults.component.html]: yes i removed the inline css

File: src/app/component/allresults/allresults.component.ts (hunk 28 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/allresults/allresults.component.ts]: okay i removed it.

File: src/app/component/dashboard/dashboard.component.html (hunk 29 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/dashboard/dashboard.component.html]: Yes i removed the inline css

File: src/app/component/quiz/quiz.component.html (hunk 30 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.html]: Yes i removed it

File: src/app/component/quiz/quiz.component.html (hunk 31 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/quiz/quiz.component.html]: yes i removed the conflict

File: src/app/component/result/result.component.html (hunk 33 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/result/result.component.html]: Yes i removed the inline css

File: src/app/component/result/result.component.html (hunk 34 above)
Comment (PrabhatJTDeveloper) [reply src/app/component/result/result.component.html]: Yes i removed the inline css

File: src/app/component/allresults/allresults.component.css [hunk 35]
```diff
@@ -0,0 +1,288 @@
...
+  margin-top: .5rem;
+}
+
+.all-result .quiz-points .points {
+  display: list-item;
+  color: #727686;
+}
+
+.all-result .quiz-points .points span {
+  margin-left: .5rem;
```
Comment (PoojanPatelJT) [inline src/app/component/allresults/allresults.component.css:279]: is it required to use span do we use class here ?

File: src/app/component/allresults/allresults.component.css [hunk 36]
```diff
@@ -0,0 +1,288 @@
...
+}
+
+.all-result .quiz-action {
+  text-align: right;
+}
+
+.all-result .quiz-action a {
+  text-transform: uppercase;
+}
```
Comment (PoojanPatelJT) [inline src/app/component/allresults/allresults.component.css:288]: Same here

File: src/app/component/dashboard/dashboard.component.html [hunk 37]
```diff
@@ -1,94 +1,40 @@
...
-                {{descriptions[imagesssindex]}}
-              </p>
-              <div class="text-center">
-                <button type="button" class="btn btn-primary" (click)="quizname()">Start Quiz</button>
-              </div>
+<div class="container">
+  <div class="row">
+    <div class="min-vh-100 d-flex align-items-center">
+      <div class="card   mt-5" id="card">
+        <div class="card-body">
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.html:5]: remove extra space from here

File: src/app/component/allresults/allresults.component.ts [hunk 38]
```diff
@@ -0,0 +1,67 @@
...
+    if (!this.userName) {
+      this.getData();
+    }
+  }
+
+  getData() {
+    let data: any = localStorage.getItem('registeruser');
+    this.userData = JSON.parse(data);
+    this.userName = this.userData.find((data:any)=>{
+      return data.id == this.questionService.getUser()
```
Comment (PoojanPatelJT) [inline src/app/component/allresults/allresults.component.ts:32]: apply prettier to the file for proper indentation

File: src/app/component/dashboard/dashboard.component.scss [hunk 39]
```diff
@@ -39,9 +37,7 @@ h5{
     height:330px;
     width:250px;
 } */
-.card{
-    height: 350px;
-}
+
 /* .card-wrapper{
```
Comment (PoojanPatelJT) [inline src/app/component/dashboard/dashboard.component.scss:41]: please remove commented code from here

**Pull request closed without merging**